modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java

```diff
@@ -197,17 +202,80 @@ public void logoutFromAdminConsole() throws Exception {
      */
     public void addDashBoard(String dashBoardTitle, String description) throws Exception {
         driver = getDriver();
-        driver.get(getBaseUrl()+"/portal/dashboards");
+
+        redirectToLocation("portal", "dashboards");
         driver.findElement(By.cssSelector("[href='create-dashboard']")).click();
         driver.findElement(By.id("ues-dashboard-title")).clear();
```

Comment:
If these Ids are being used in multiple places and especially multiple classes, add them as constants and reuse them without declaring everywhere.